### PR TITLE
Fix single maintenance-inner-central tile designation

### PR DESCRIPTION
**Pull request by pgmzeta**
## About the PR 
Replaces one EVA tile with an inner-maintenance tile on oshan and oshan-xmas

## Why's this needed? 
Founds by radstorms affect the EVA tile, but not maintenance tiles.
Fixes https://github.com/goonstation/goonstation/issues/6922

## Notes
* I don't know why the changes to each map resulted in different diffs (???)
* First PR here, please double-check my work <3

### CODE REVIEW

**Comment by ZeWaka** [comment]
It resulted in different diffs due to how the map files are encoded.

**Review by ZeWaka — CHANGES_REQUESTED** [review]
`/area/station/maintenance/inner` is not a valid area, per the error in the runtime checker

File: maps/oshan.dmm [hunk 1]
```diff
@@ -44473,7 +44473,7 @@
 /area/station/crew_quarters/radio/lab)
 "ciK" = (
 /turf/simulated/floor/plating/random,
-/area/station/storage/eeva)
+/area/station/maintenance/inner)
```
Comment (ZeWaka) [inline maps/oshan.dmm:44476]: still didn't change it

Comment (pgmzeta) [reply maps/oshan.dmm:44476]: thank you 🙏

**Pull request merged**